### DISCUSSION TITLE
@eslint/core — ESLint without JavaScript

### DISCUSSION THREAD

**Discussion by vanyauhalin** [eslint/eslint#16331] [Ideas] [answered]
Throughout it is existence, ESLint has been focused on working with JavaScript, but thanks to the community, it has become possible to work with, for example, [JSON](https://github.com/ota-meshi/eslint-plugin-jsonc) or [YAML](https://github.com/ota-meshi/eslint-plugin-yml) files. What I mean is that I would like to use ESLint in projects that may not have JavaScript code at all, but has languages that can be fixed with ESLint.

```
@eslint/core            ESLint core utils, functions
@eslint/js              ESLint basic rules for JavaScript
@eslint/base or eslint  ESLint we are currently using
├─ @eslint/core
└─ @eslint/js
```

**Comment by kecrily** [comment, marked as answer]
I think you could try submitting an RFC

https://github.com/eslint/rfcs
